bin/plugin/commands/performance.js

```diff
@@ -186,14 +186,24 @@ function curateResults( testSuite, results ) {
  *
  * @param {string} testSuite                Name of the tests set.
  * @param {string} performanceTestDirectory Path to the performance tests' clone.
+ * @param {string} runKey                   Unique identifier for the test run, e.g. `branch-name_post-editor_run-3`.
  *
  * @return {Promise<WPPerformanceResults>} Performance results for the branch.
  */
-async function runTestSuite( testSuite, performanceTestDirectory ) {
+async function runTestSuite( testSuite, performanceTestDirectory, runKey ) {
 	await runShellScript(
 		`npm run test:performance -- packages/e2e-tests/specs/performance/${ testSuite }.test.js`,
 		performanceTestDirectory
 	);
+	const resultsFile = path.join(
+		performanceTestDirectory,
+		`packages/e2e-tests/specs/performance/${ testSuite }.test.results.json`
+	);
+	fs.mkdirSync( '/home/runner/__test-results', { recursive: true } );
+	fs.copyFileSync(
+		resultsFile,
+		`/home/runner/__test-results/${ runKey }.results.json`
```

Comment:
we probably need to change this directory. I choose it because I knew it would work in Github Actions, but I didn't verify that it works in Docker when running the tests locally. will do so before merge.